netstandard/src/netstandard.csproj

```diff
@@ -40,26 +40,36 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='xamarin.ios'">
     <ProjectReference Include="..\..\platforms\$(TargetGroup)\OpenTK-1.0.csproj" />
+    <ProjectReference Include="..\..\platforms\$(TargetGroup)\System.Numerics.Vectors.csproj" />
     <ProjectReference Include="..\..\platforms\$(TargetGroup)\System.Web.Services.csproj" />
     <ProjectReference Include="..\..\platforms\$(TargetGroup)\Xamarin.iOS.csproj" />
+    <ProjectReference Include="..\..\platforms\$(TargetGroup)\Facades\*.csproj" />
```

Comment:
We should probably consider using *.csproj for all the references and just control them via the presence in `platforms\$(TargetGroup)`.